.connector-store/meta.json

```diff
@@ -16,6 +16,20 @@
         "inbound-endpoint"
     ],
     "releases": [
+        {
+            "tagName": "v1.2.5",
+            "products": [
+                "MI 1.2.0",
```

Comment:
Is this order shown in the store? if so, please reorder this.

Reply:
Fixed in 3d1aff1681870c1de4d5b2b31ebabe8be2c684f7